tasks/db_deploy/orca_sql.py

```diff
@@ -584,3 +584,17 @@ def drop_drdbo_role_sql() -> TextClause:
         DROP ROLE IF EXISTS drdbo_role;
     """
     )
+
+
+def add_multipart_chunksize_sql() -> TextClause:
+    """
+    SQL that adds the multipart_chunksize_mb column to orca_files.
+
+    Returns: SQL for adding multipart_chunksize_mb.
+    """
+    return text(
+        """
+        ALTER TABLE table_name
```

Comment:
change table_name to be the recovery table you want to add the column to. Possible the recovery_job table so it would like be set at the granule/collection level?

Reply:
You will also want to add the column into the table definition of the table you are altering so that a fresh create will have that column.

Reply:
I believe I did the later already... recovery_file_table_sql, yes? Going with recovery_file for the former.